editor/package.json

```diff
@@ -27,11 +27,11 @@
-    "performance-test": "NODE_OPTIONS=--max_old_space_size=8192 PERFORMANCE='true' webpack-dev-server --define process.env.NODE_ENV=\"'production'\" --define process.env.REACT_APP_ENVIRONMENT_CONFIG=\"'development'\"  --progress --define process.env.REACT_APP_AUTH0_CLIENT_ID=\"'$AUTH0_CLIENT_ID'\" --define process.env.REACT_APP_AUTH0_ENDPOINT=\"'$AUTH0_ENDPOINT'\" --define process.env.REACT_APP_AUTH0_REDIRECT_URI=\"'$AUTH0_REDIRECT_URI'\" --define process.env.REACT_APP_COMMIT_HASH=\"'`git rev-parse HEAD`'\" --define process.env.GOOGLE_WEB_FONTS_KEY=\"'$GOOGLE_WEB_FONTS_KEY'\"",
-    "move-fast-and-break-things-hot": "NODE_OPTIONS=--max_old_space_size=8192 HOT='true' webpack-dev-server --progress --define process.env.REACT_APP_AUTH0_CLIENT_ID=\"'$AUTH0_CLIENT_ID'\" --define process.env.REACT_APP_AUTH0_ENDPOINT=\"'$AUTH0_ENDPOINT'\" --define process.env.REACT_APP_AUTH0_REDIRECT_URI=\"'$AUTH0_REDIRECT_URI'\" --define process.env.REACT_APP_COMMIT_HASH=\"'`git rev-parse HEAD`'\" --define process.env.GOOGLE_WEB_FONTS_KEY=\"'$GOOGLE_WEB_FONTS_KEY'\"",
-    "production": "WEBPACK_MODE='production' NODE_OPTIONS=--max_old_space_size=6656 webpack --define process.env.REACT_APP_ENVIRONMENT_CONFIG=\"'production'\" --define process.env.REACT_APP_AUTH0_CLIENT_ID=\"'$AUTH0_CLIENT_ID'\" --define process.env.REACT_APP_AUTH0_ENDPOINT=\"'$AUTH0_ENDPOINT'\" --define process.env.REACT_APP_AUTH0_REDIRECT_URI=\"'$AUTH0_REDIRECT_URI'\" --define process.env.REACT_APP_COMMIT_HASH=\"'$UTOPIA_SHA'\" --define process.env.GOOGLE_WEB_FONTS_KEY=\"'$GOOGLE_WEB_FONTS_KEY'\"",
-    "staging": "WEBPACK_MODE='staging' NODE_OPTIONS=--max_old_space_size=6656 webpack --define process.env.REACT_APP_ENVIRONMENT_CONFIG=\"'staging'\" --define process.env.REACT_APP_AUTH0_CLIENT_ID=\"'$AUTH0_CLIENT_ID'\" --define process.env.REACT_APP_AUTH0_ENDPOINT=\"'$AUTH0_ENDPOINT'\" --define process.env.REACT_APP_AUTH0_REDIRECT_URI=\"'$AUTH0_REDIRECT_URI'\" --define process.env.REACT_APP_COMMIT_HASH=\"'$UTOPIA_SHA'\" --define process.env.GOOGLE_WEB_FONTS_KEY=\"'$GOOGLE_WEB_FONTS_KEY'\"",
+    "move-fast-and-break-things": "REACT_APP_ENVIRONMENT_CONFIG=development REACT_APP_AUTH0_CLIENT_ID=$AUTH0_CLIENT_ID REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=8192 webpack-dev-server --progress",
+    "performance-test": "NODE_ENV=production REACT_APP_ENVIRONMENT_CONFIG=development REACT_APP_AUTH0_ENDPOINT=$AUTH0_ENDPOINT REACT_APP_AUTH0_REDIRECT_URI=$AUTH0_REDIRECT_URI REACT_APP_COMMIT_HASH=`git rev-parse HEAD` NODE_OPTIONS=--max_old_space_size=8192 PERFORMANCE='true' webpack-dev-server --progress",
```

Comment:
This one is missing `REACT_APP_AUTH0_CLIENT_ID`

Reply:
fixed!